docs/source/highlights.md

```diff
@@ -49,7 +49,7 @@ new_img = affine(image, spatial_size=(300, 400), mode='bilinear')
 ```
 
 ### 4. Randomly crop out batch images based on positive/negative ratio
-Medical image data volume may be too large to fit into GPU memory. A widely-used approach is to randomly draw small size data samples during training. MONAI currently provides uniform random sampling strategy as well as class-balanced fixed ratio sampling which may help stabilize the patch-based training process.
+Medical image data volume may be too large to fit into GPU memory. A widely-used approach is to randomly draw small size data samples during training. MONAI currently provides many general random sampling strategies as well as class-balanced fixed ratio sampling which may help stabilize the patch-based training process.
```

Comment:
remove 'many', and 'as well as' -> 'including'

Reply:
changed in the latest patch.
Thanks.